website/docs/cloud-docs/agents/request-forwarding.mdx

```diff
@@ -50,6 +51,13 @@ the agent starts, you will see the following log messages:
 These log messages indicate that the agent has successfully connected to HCP Terraform
 and is ready to start forwarding requests.
 
+### (Optional) Accept ingress jobs needed for Private VCS access
+By default, HCP Terraform will try to clone a repository using its own infrastructure.
+For Private repository access to work, Configuration Version(CV) ingress needs to happen through the agent.
+For this ingress job type to be accepted, agent must be started with `-accept` containing `ingress`.
```

Comment:
Can you provide an example of the command, or how else folks are supposed to use the `-accept` flag?